poly/src/domain/radix2/fft.rs

```diff
@@ -213,6 +232,22 @@ const MIN_CHUNK_SIZE_FOR_PARALLELIZATION: usize = 2048;
 #[cfg(feature = "parallel")]
 const LOG_ROOTS_OF_UNITY_PARALLEL_SIZE: u32 = 7;
 
+// Maximum stride allowed for the root vector, before re-cache aligning it.
+// This won't cache align if we've reached the ROOT_STOP_RESIZING threshold.
+// TODO: Better understand how this should be optimized
+// (according to cache lines, and hardware prefetchers)
+const MAX_ROOT_STRIDE: usize = 2;
+
+// Once the size of the cache aligned roots is below this number, stop re-aligning it.
+// TODO: Figure out how we can make this depend on field size & system cache size.
```

Comment:
We can make it depend on field size by figuring out the field size as `F::characteristic() * F::degree()`

Reply:
But its a constant, so it would have to go in the FftField params right?

Reply:
Or can you have constants depend on a template parameter?

Reply:
Ah you're right, they're not `const fn`... I'm not sure we need it to be actually const; the compiler should be able to perform constant folding